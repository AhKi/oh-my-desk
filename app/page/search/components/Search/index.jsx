import React from 'react';
import SearchInputContainer from '../../containers/SearchInputContainer';
import SearchList from '../SearchList';
import SearchMenu from '../SearchMenu';
import './Search.scss';

function Search() {
  return (
    <div className="Search">
      <SearchMenu />
      <SearchInputContainer />
      <SearchList />
    </div>
  );
}

export default Search;
