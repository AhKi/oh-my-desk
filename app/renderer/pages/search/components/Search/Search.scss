.Search {
  display: flex;
  height: 100%;
}

.Search__content {
  display: flex;
  flex-direction: column;
  flex: 1;
}
