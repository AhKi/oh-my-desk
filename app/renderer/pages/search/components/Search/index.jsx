import React from 'react';
import SearchInputContainer from '../../containers/SearchInputContainer';
import SearchListContainer from '../../containers/SearchListContainer';
import SearchMenuContainer from '../../containers/SearchMenuContainer';
import './Search.scss';

class Search extends React.Component {
  render() {
    return (
      <div className="Search">
        <SearchMenuContainer />
        <SearchInputContainer />
        <SearchListContainer />
      </div>
    );
  }
}

export default Search;
