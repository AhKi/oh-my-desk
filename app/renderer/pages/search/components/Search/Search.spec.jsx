import React from 'react';
import { shallow } from 'enzyme';
import Search from './index';

describe('Test Search Component', () => {
  it('should match to snapshot', () => {
    const wrapper = shallow(<Search />);

    expect(wrapper).toMatchSnapshot();
  });
});
