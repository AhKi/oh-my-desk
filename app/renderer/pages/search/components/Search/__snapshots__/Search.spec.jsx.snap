// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Search Component should match to snapshot 1`] = `
<div
  className="Search"
>
  <Connect(toJS(SearchMenu)) />
  <Connect(toJS(SearchInput)) />
  <Connect(toJS(SearchList)) />
</div>
`;
